content/en/_index.html

```diff
@@ -19,7 +19,7 @@ <h2 class="mt-4">
   A distributed, reliable key-value store for the most critical data of a distributed system
 </h2>
 <div class="mt-5 mx-auto">
-	<a class="btn btn-lg btn-primary mr-3 mb-4" href="/docs/latest/" data-proofer-ignore>
+	<a class="btn btn-lg btn-primary mr-3 mb-4" href="/docs/{{< param versions.latest >}}/" data-proofer-ignore>
```

Comment:
do we still need `data-proofer-ignore` if we're using a hugo generated link here? (it was added to make the link checker happy)

Reply:
Oh! I was sure that I had removed that. Thanks, I'll take it out.

Reply:
Oh, I did it for `master` via #221.

Reply:
Thanks for catching it! Fixed